src/puppy/platforms/linux/platform.nim

```diff
@@ -31,6 +31,10 @@ proc fetch*(req: Request): Response {.raises: [PuppyError].} =
 
   let curl = easy_init()
 
+  if req.insecure:
```

Comment:
Wow this is so easy, I whish other platforms where like this.

Reply:
Same. Sadly, no other operating system can compare to linux (GNU/Linux for anyone that'll get mad).